packages/registrar_client/README.md

```diff
@@ -0,0 +1,81 @@
+# Registrar Client
+
+This package provides a client for interacting with the TFGrid v4 Node Registrar.
+
+## Prerequisites
+
+- node 20.10.0 or higher
+- npm 10.2.3 or higher
+
+## Installation
+
+1. **Clone the Repository**
+
+   ```bash
+   git clone https://github.com/threefoldtech/tfgridv4-sdk-ts
+   cd tfgridv4-sdk-ts
+   ```
+
+2. **Navigate to the Package Directory**
+
+   ```bash
+   cd packages/registrar_client
+   ```
+
+3. **Install Dependencies**
+
+   ```bash
+   yarn install
+   ```
+
+4. **Build the Package**
+
+   ```bash
+   yarn build
+   ```
+
+## Getting Started
+
+To initialize the Registrar Client, you need to provide the base url of registrar and your private key. Here is an example:
```

Comment:
Can you mention how to create a private key? and if there is any limitation on the key type please mention it